src/fios.h

```diff
@@ -113,7 +113,7 @@ class FileList {
 	 */
 	inline FiosItem *Append()
 	{
-		return this->files.Append();
+		return &*Extend(this->files, 1);
```

Comment:
Ew. Looks like FileList is itself a pseudo-SmallVector - should be "converted" ?